site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -312,6 +313,33 @@ If managed FusionAuth hosting does not meet your needs, open a link:/contact/[co
 
 For instance, if you need longer retention of database backups, we're happy to discuss that with you.
 
+== SLAs and Availability
+
+We know that availability of your auth system is critical. Below are some of the steps FusionAuth takes to ensure that FusionAuth cloud systems are available. You can find information about service level agreements (SLAs) in the link:/license/[FusionAuth license] and link:/license-faq/[license FAQ].
```

Comment:
Should we capitalize `cloud` here since it is a product name? "FusionAuth Cloud"